Dockerfile

```
@@ -0,0 +1,25 @@
+FROM registry.fedoraproject.org/fedora-toolbox:38
+RUN dnf update -y && \
+    dnf install -y \
+    	git \
+    	wine \
+	wine-mono \
+	python3 \
+	msitools \
+	python3-simplejson \
+	python3-six \
+	cmake \
+	ninja-build \
+	make \
+	samba \
+	libunwind && \
+    dnf clean all && \
+    mkdir /opt/msvc/ /build
+
```

doesn't work without this for me
```suggestion

RUN git config --global --add safe.directory /build
```

---

I had to do the same iirc